### PR TITLE
Ported Ritual of the Feathered Earth to BM2.

### PR DESCRIPTION
**Pull request by keraldi**

### CODE REVIEW

**Comment by TehNut** [comment]
I feel like a more user-friendly implementation of this would be to handle the actual effect in a damage event and cancelling it if it's fall damage while they're in range.

This current implementation is just constantly setting fall damage to 0 every second, even for entities which aren't moving, which just seems wrong.

**Comment by keraldi** [comment]
Done.

File: gradle.properties [hunk 1]
```diff
@@ -2,7 +2,7 @@ mod_name=BloodMagic
 package_group=com.wayoftime.bloodmagic
 mod_version=2.3.3
 mc_version=1.12.2
-forge_version=14.23.2.2611
+forge_version=14.23.5.2768
```
Comment (TehNut) [inline gradle.properties]: Is this necessary?

**Review by TehNut — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (keraldi) [reply gradle.properties]: My bad.

**Review by TehNut — APPROVED** [review]

**Pull request merged**